cmd/main.go

```diff
@@ -1,17 +1,35 @@
 package main
 
 import (
-	"log"
 	"net/http"
 
+	"github.com/gin-gonic/gin"
 	"github.com/tech-pool/takemetotheshop-api/api/registration"
 )
 
 const address string = ":8080"
```

Comment:
Is it possible to take the port number from the environment variable instead of hardcoding it as a string?

Reply:
done